CODEOWNERS

```
@@ -0,0 +1 @@
+* @mangelajo @tpantelis @Oats87
```

```suggestion
* @submariner-io/submariner-core 
```

I think this is better, in that way if we modify the team, we don't need to replicate the change across all repos.

https://github.com/orgs/submariner-io/teams

---

Yeah, that makes sense. I was thinking about this line from the K8s docs:

> We use aliases for groups instead of GitHub Teams, because changes to GitHub Teams are not publicly auditable.

https://github.com/kubernetes/community/blob/7d2ebad43cde06607cde3d55e9eed4bb08a286a9/contributors/guide/owners.md#owners_aliases

Ie, the upstream process we're trying to copy explicitly lists usernames vs using GitHub teams for security auditing reasons.

https://github.com/kubernetes/kubernetes/blob/master/OWNERS

---

ah, ok, makes sense

---

Sorry for the delay!

---

No worries, thanks! It seems like things worked! :smile: